[CI] Failure in CCSDuelIT.testFieldCollapsingOneClusterHasNoResults

**Build scan**:
https://gradle-enterprise.elastic.co/s/oict634dq5vga

**Repro line**:
./gradlew ':qa:multi-cluster-search:remote-cluster' --tests "org.elasticsearch.search.CCSDuelIT.testFieldCollapsingOneClusterHasNoResults" \
  -Dtests.seed=700B200D94EBA284 \
  -Dtests.security.manager=true \
  -Dtests.locale=es-CU \
  -Dtests.timezone=Asia/Singapore \
  -Druntime.java=11

**Reproduces locally?**:
No

**Applicable branches**:
7.x

**Failure history**:
Couldn't find more recent failures

There seems to be a sporadic intialization assertion problem here:

**Failure excerpt**:
java.lang.AssertionError: expected:<1> but was:<2>
	at __randomizedtesting.SeedInfo.seed([700B200D94EBA284:2025134D3E63EADA]:0)
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at org.elasticsearch.search.CCSDuelIT.indexDocuments(CCSDuelIT.java:221)
	at org.elasticsearch.search.CCSDuelIT.lambda$init$0(CCSDuelIT.java:141)
	at com.carrotsearch.randomizedtesting.RandomizedContext.runWithPrivateRandomness(RandomizedContext.java:187)
	at com.carrotsearch.randomizedtesting.RandomizedContext.runWithPrivateRandomness(RandomizedContext.java:211)
	at org.elasticsearch.search.CCSDuelIT.init(CCSDuelIT.java:139)

--- a/qa/multi-cluster-search/src/test/java/org/elasticsearch/search/CCSDuelIT.java
+++ b/qa/multi-cluster-search/src/test/java/org/elasticsearch/search/CCSDuelIT.java
@@ -10,6 +10,7 @@ package org.elasticsearch.search;
 
 import com.carrotsearch.randomizedtesting.RandomizedContext;
 import com.carrotsearch.randomizedtesting.annotations.TimeoutSuite;
+
 import org.apache.lucene.search.join.ScoreMode;
 import org.apache.lucene.util.TimeUnits;
 import org.elasticsearch.action.ActionListener;
@@ -74,6 +75,7 @@ import org.elasticsearch.search.suggest.phrase.PhraseSuggestionBuilder;
 import org.elasticsearch.search.suggest.term.TermSuggestion;
 import org.elasticsearch.search.suggest.term.TermSuggestionBuilder;
 import org.elasticsearch.test.NotEqualMessageBuilder;
+import org.elasticsearch.test.hamcrest.ElasticsearchAssertions;
 import org.elasticsearch.test.rest.ESRestTestCase;
 import org.junit.AfterClass;
 import org.junit.Before;
@@ -206,8 +208,7 @@ public class CCSDuelIT extends ESRestTestCase {
         assertTrue(bulkProcessor.awaitClose(30, TimeUnit.SECONDS));
 
         RefreshResponse refreshResponse = restHighLevelClient.indices().refresh(new RefreshRequest(INDEX_NAME), RequestOptions.DEFAULT);
-        assertEquals(0, refreshResponse.getFailedShards());
-        assertEquals(numShards, refreshResponse.getSuccessfulShards());
+        ElasticsearchAssertions.assertNoFailures(refreshResponse);
     }
 
     private static IndexRequest buildIndexRequest(String id, String type, String questionId) {

